pygeos/_geometry.pyx

```diff
@@ -1,36 +1,38 @@
 # distutils: define_macros=GEOS_USE_ONLY_R_API
 
-from cpython cimport PyObject
 cimport cython
+from cpython cimport PyObject
```

Comment:
@caspervdw I'm getting different ordering on `master` when I run `pre-commit run --files pygeos/_geometry.pyx`

```
from cpython cimport PyObject

cimport cython
from cython cimport view
```

So it fails the linting check...
(using latest version of `isort`, 5.8.0 but also happened on 4.3 prior to upgrading it)

Not seeing what would cause this to be different locally; CI seems to be passing OK.  Any ideas other than wiping and reinstalling env?

Reply:
If you are using pre-commit, it should already use a separate isolated env (so wiping the env shouldn't help normally)

For me that is passing:

```
(geo-dev) joris@joris-XPS-13-9380:~/scipy/repos/pygeos$ pre-commit run --files pygeos/_geometry.pyx
black................................................(no files to check)Skipped
flake8...............................................(no files to check)Skipped
isort....................................................................Passed
```